aiida/post-init.sh

```diff
@@ -20,8 +20,9 @@ aiida_profile="aiida_renku"
 institution="RenkuLab"
 
 project_dir="$(pwd)"
-repo_dir="${project_dir}/repo"
+repo_dir="${project_dir}/aiida_data"
 mkdir "$repo_dir"
+echo "$repo_dir" >> .gitignore
```

Comment:
This will add `repo_dir` to the `.gitignore` every time the session starts; it also makes the repo dirty which isn't so nice for the user. I think you can simply add `aiida_data` to the `.gitignore` and get rid of this line.

Reply:
So, `post-init.sh` is run every time the session starts, not just when the project is created? I just verified that, and that's an issue, as the AiiDA profile should also only be created once. I could add a check for that in the `post-init.sh` file, but that seems a bit hacky. Is there any other place to put such setup steps that come after the docker image build, but are only run once on project creation, rather than at every session startup?

Regarding adding `aiida_data` to the `.gitignore`, good point, I'll add it there directly via its relative path, as you suggested.

Reply:
> I just verified that, and that's an issue, as the AiiDA profile should also only be created once. I could add a check for that in the post-init.sh file, but that seems a bit hacky. Is there any other place to put such setup steps that come after the docker image build, but are only run once on project creation, rather than at every session startup?

It might not be a bad idea to still run this at each session startup in case the first time it failed for some reason. You can easily make the action idempotent by just checking if the target profile exists. Something like:
```
profile_name=some-profile
verdi profile show $profile_name 2> /dev/null || verdi profile setup ....
```

Reply:
I'll resolve it like this, thanks, @sphuber!